eureka-client/src/main/java/com/netflix/discovery/converters/jackson/EurekaJsonJacksonCodec.java

```diff
@@ -37,7 +39,7 @@ public EurekaJsonJacksonCodec() {
     public EurekaJsonJacksonCodec(final KeyFormatter keyFormatter, boolean compact) {
```

Comment:
It's in a public constructor and there are already cases of extensions of the Codec classes.